js/node/src/inference_session_wrap.cc

```diff
@@ -203,7 +203,8 @@ Napi::Value InferenceSessionWrap::Run(const Napi::CallbackInfo &info) {
     Napi::Object result = Napi::Object::New(env);
 
     for (size_t i = 0; i < outputIndex; i++) {
-      result.Set(outputNames_[i], OrtValueToNapiValue(env, outputValues[i]));
+      result.Set(outputNames_[i],
+                 OrtValueToNapiValue(env, std::move(std::make_unique<Ort::Value>(std::move(outputValues[i])))));
```

Comment:
Why this line of code needs so many std::move?

Reply:
1. Move out the element from std::vector
2. Create unique_ptr
3. Move unique_ptr owner (Let `OrtValueToNapiValue` manage it